span-normalizer/span-normalizer/src/test/java/org/hypertrace/core/spannormalizer/fieldgenerators/SqlFieldsGeneratorTest.java

```diff
@@ -78,13 +79,18 @@ public void testPopulateOtherFields() {
     SqlFieldsGenerator sqlFieldsGenerator = new SqlFieldsGenerator();
 
     Event.Builder eventBuilder = Event.newBuilder();
-    Map<String, AttributeValue> map = Maps.newHashMap();
-    map.put(OTelDbSemanticConventions.DB_SYSTEM.getValue(),
-        AttributeValue.newBuilder().setValue(OTelDbSemanticConventions.MYSQL_DB_SYSTEM_VALUE.getValue()).build());
-    map.put(OTelDbSemanticConventions.DB_CONNECTION_STRING.getValue(),
-        AttributeValue.newBuilder().setValue("jdbc:mysql://mysql:3306/shop").build());
+
+    Map<String, AttributeValue> map = Map.of(
+        OTelDbSemanticConventions.DB_SYSTEM.getValue(),
+        AttributeValue.newBuilder().setValue(OTelDbSemanticConventions.MYSQL_DB_SYSTEM_VALUE.getValue()).build(),
+        OTelDbSemanticConventions.DB_CONNECTION_STRING.getValue(),
+        AttributeValue.newBuilder().setValue("jdbc:mysql://mysql:3306/shop").build(),
```

Comment:
For value of `db.connection_string`, can you replace this with  - `Server=shopdb.example.com;Database=ShopDb;Uid=billing_user;TableCache=true;UseCompression=True;MinimumPoolSize=10;MaximumPoolSize=50;` ?
Ref : https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#mysql

Reply:
done